### PR TITLE
MNTOR-1675: Welcome scan progress label

### PR DESCRIPTION
**Pull request by flozia**
# References: 
Jira: [MNTOR-1675](https://mozilla-hub.atlassian.net/browse/MNTOR-1675)
Figma: https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=1398-73778&t=LMpwec8NLhVLuzS7-4

# Description

This add the correct labels to the progress bar in the “find exposures” welcome step.

# Screenshot

https://github.com/mozilla/blurts-server/assets/13835474/ae118c7b-db6f-491f-bab1-9fc698708943

# How to test

Same steps as for https://github.com/mozilla/blurts-server/pull/3222

### CODE REVIEW

File: src/db/tables/breaches.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ async function getAllBreaches() {
...
 }
 
+/**
+ * Get all count from "breaches" table
+ *
+ * @returns Count of all records from "breaches" table
+ */
+async function getAllBreachesCount() {
+  const breachesCount = await knex('breaches')
+    .count({ count: "*" })
```
Comment (rhelmer) [inline src/db/tables/breaches.js:28]: This table is not very large, but if it became a problem we could consider doing an index-only scan (e.g. if the index is likely to be significantly smaller than the table).

File: src/db/tables/breaches.js [hunk 2]
```diff
@@ -18,6 +18,20 @@ async function getAllBreaches() {
...
+    .first()
+    .then(result => result?.count)
+  // @ts-ignore The above query is returning a string
+  return parseInt(breachesCount, 10)
```
Comment (rhelmer) [inline src/db/tables/breaches.js:32]:
```suggestion
  return parseInt(breachesCount?.toString() || "", 10)
```

**Review by rhelmer — APPROVED** [review]
I think we could avoid the `@ts-ignore`, this should return `NaN` if the empty string is passed to `parseInt()`

File: src/db/tables/breaches.js (hunk 1 above)
Comment (flozia) [reply src/db/tables/breaches.js]: Good suggestion! Made the change in [bd3a6df](https://github.com/mozilla/blurts-server/pull/3233/commits/bd3a6df8c2ed1e6c2e59780192ad8b0f2fc822d6).

File: src/db/tables/breaches.js (hunk 2 above)
Comment (flozia) [reply src/db/tables/breaches.js]: Thanks — changed in [bd3a6df](https://github.com/mozilla/blurts-server/pull/3233/commits/bd3a6df8c2ed1e6c2e59780192ad8b0f2fc822d6).

**Pull request merged**